.chloggen/process_originevents.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: process
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: add process.executable.origin_referrer_url and process.executable.origin_url
```

Comment:
are those necessary - could similar `file` attributes being introduced in https://github.com/open-telemetry/semantic-conventions/pull/1430 be used instead? if not, why?